pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.github.invictum</groupId>
     <artifactId>serenity-reportportal-integration</artifactId>
-    <version>1.6.0-SNAPSHOT</version>
+    <version>1.6.1-SNAPSHOT</version>
```

Comment:
Usually we don't need to manually increment snapshot versions, it is automated via release plugin.
Any special reason to do it in current PR?